api/types/swarm/task.go

```diff
@@ -65,6 +65,11 @@ type TaskSpec struct {
 	// ForceUpdate is a counter that triggers an update even if no relevant
 	// parameters have been changed.
 	ForceUpdate uint64
+
+	Runtime RuntimeType `json:",omitempty"`
+	// TODO (ehazlett): this should be removed and instead
+	// use struct tags (proto) for the runtimes
+	RuntimeData []byte `json:",omitempty"`
```

Comment:
Don't forget to update the swagger.yml with the new fields; https://github.com/docker/docker/blob/9f9cc221bdb0a919d6ed28bdfaffa9ae2fe3a5fd/api/swagger.yaml#L1894-L2116